server/www/templates/partials/user-menu.html

```diff
@@ -1,6 +1,6 @@
 <div class="user-admin logged-out" ng-hide="currentUser">
     <a ng-href="/login" class="user-login" translate>nav.login</a>
-    <a ng-href="/register" class="user-register" translate>nav.register</a>
+    <a ng-href="/register" class="user-register" translate ng-show="!site.private">nav.register</a>
```

Comment:
Similar - maybe put this in a function so its `canRegister` .. more understandable then.